mag2exp/fmr.py

```diff
@@ -0,0 +1,116 @@
+"""FMR submodule.
+
+Module for calculation of Ferromagnetic Resonance related
+quantities.
+"""
+
+from typing import Optional, Tuple
+
+import discretisedfield as df
+import hvplot.xarray  # noqa
+import micromagneticdata as micd
+import numpy as np
+import scipy.fft as fft
+import xarray as xr
+
+
+def fmr(
+    drive: micd.Drive, init_field: Optional[df.Field] = None
+) -> Tuple[xr.DataArray, xr.DataArray]:
```

Comment:
Are you sure `xr.DataArray` is the correct type? Numpy e.g. has a separate `numpy.typing` submodule.

Reply:
`xarray` has no typing submodule but I have checked the following

```python
>>> type(power)
xarray.core.dataarray.DataArray
>>> xr.DataArray
xarray.core.dataarray.DataArray
>>> type(power) == xr.DataArray
True
```